R/proxy.r

```diff
@@ -99,7 +99,7 @@ sqlite_ld_proxies <- function(rsids, dbfile, tag_r2)
 	numid <- gsub("rs", "", rsids) %>% paste(collapse=",")
 	query <- paste0("SELECT DISTINCT * FROM tags WHERE SNP_A IN (", numid, ")")
 	ld <- RSQLite::dbGetQuery(conn, query) %>% 
-		dplyr::as_tibble() %>%
+		dplyr::as_tibble(.data, .name_repair="minimal") %>%
```

Comment:
Recommend deleting the `.data, ` as it's not needed (and I think this is what's causing the error in building guide.Rmd).